sources/web/datalab/polymer/components/file-browser/file-browser.ts

```diff
@@ -803,8 +846,15 @@ class FileBrowserElement extends Polymer.Element implements DatalabPageElement {
         }
       }, this._fileListRefreshInterval);
     }
-    // Now refresh the list once.
-    this._fetchFileList();
+    // Refresh the list and update the window location.
+    this._pathHistoryIndexChanged();
```

Comment:
If you're calling this method to get the convenience of calling both `_fetchFileList` and `_setFileParamToCurrentFile`, I'd inline the two calls here, since that method has different semantics, and it could change at any point.